src/main.ts

```diff
@@ -55,11 +56,15 @@ export async function activate(context: vscode.ExtensionContext) {
 		},
 	};
 
+	let commandObserver = new CommandObserver();
 	const installationStart = Date.now();
 	serverdownloader.getOrDownloadServer().then(e => {
+		vscode.window.showErrorMessage(e);
 		const installationComplete = Date.now();
 		let serverOptions = generateServerOptions(e);
 		languageClient = new SqlOpsDataClient(Constants.serviceName, serverOptions, clientOptions);
+		commandObserver._client = languageClient;
```

Comment:
What do we need this for? Command observer doesn't look like something that would need language client. Also underscore usually indicates private member, you should be just randomly adding ```_``` members to classes whenever you want to...

Reply:
I don't think I need this anymore, will pass client as part of command and will get rid of this.